catalog/core/catalog-core-eventcommands/src/main/java/ddf/catalog/pubsub/command/ListCommand.java

```diff
@@ -69,10 +76,24 @@ protected Object doExecute() throws Exception {
             console.println();
             console.print(CYAN_CONSOLE_COLOR);
             console.print(SUBSCRIPTION_ID_COLUMN_HEADER);
+            console.print("\t\t| ");
+            console.print(CALLBACK_COLUMN_HEADER);
             console.println(DEFAULT_CONSOLE_COLOR);
 
-            for (String subscriptionId : subscriptionIds) {
-                console.println(subscriptionId);
+            for (Map.Entry<String, ServiceReference<Subscription>> entry : subscriptionIds.entrySet()) {
```

Comment:
Do you think we should extract the code that prints out each subscription to a `private` method (e.g., `printSubscription(Map.Entry<String, ServiceReference<Subscription>> entry)`) and do this instead?

```
subscriptionIds.entrySet().forEach(this::printSubscription);
```